theme/icons/image_placeholder.svg

```diff
@@ -1 +1,8 @@
-<svg width="800" height="380" viewBox="0 0 800 380" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><rect fill="#F7F7F7" width="800" height="380" rx="4"/><text font-family="Roboto-Regular, Roboto" font-size="36" fill="#A1B0C2"><tspan x="253.188" y="200">Uploading image…</tspan></text></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 700 250">
```

Comment:
What do those changes do?

Reply:
From the ticket description:
> We should also create placeholder that have different width/height ratio to not take as much vertical space when it will be on 100% of the width.

Reply:
I will change the file to optimized one.